src/breadcrumbs/_breadcrumbs.scss

```diff
@@ -105,10 +105,18 @@
   .govuk-c-breadcrumbs__link {
     @include govuk-focusable-fill;
 
+    // Override link colour to use text colour
+    //
+    // GOV.UK Template includes a specific a:link:focus selector designed to
+    // make unvisited links a slightly darker blue when focussed, so we need to
+    // override the text colour for that combination of selectors so so that
+    // the skip link does not end up with dark blue text when focussed. (1)
     &:link,
     &:visited,
     &:hover,
-    &:active {
+    &:active,
```

Comment:
Could we break these out into their own sections and do something like

```scss
.class {
  &:link,
  &:visited,
  &:hover,
  &:active {
    color: $govuk-text-colour;
  }

  if ($with-govuk-template) {
    ... overrides here
  }
}

Reply:
I like this in principle, but not sure how it'll work in practise – especially for users that are not compiling the Sass themselves. I'm not sure what we'd do for the `dist` build, for example?

Also, I'm not sure that we have the tooling in place to have confidence in this when introducing yet more complexity into the builds. In the example above we'd need to set `color: $govuk-text-colour;` in two places, and I could e.g. foresee one being changed without the other which could lead to inconsistent and hard to reproduce problems.

Reply:
Pushed [a branch to show what this might look like in practise](https://github.com/alphagov/govuk-frontend/compare/override-link-focus-in-govuk-template...link-focus-compatibility-variation?expand=1)

Reply:
For general builds we would set these to on by default, as is the same as your current code.

Reply:
Low-fi is just to make sure each selector that overrides for govuk-template has a comment next to it, which most of these do?